### PR TITLE
Update keywords and other minor changes

### PR DESCRIPTION
**Pull request by KristjanESPERANTO**
I have looked at the code and find the repo very well done! :clap:

Nevertheless, I have a few suggestions for changes. Hopefully most of them are self-explanatory. Otherwise, the commit descriptions may help.

There are no functional changes.

The keywords in the `package.json` are actually the most important to me, as they are missing for this [module list](https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/).

### CODE REVIEW

**Review by CFenner — APPROVED** [review]
👍 Thanks for the fixes!

**Comment by KristjanESPERANTO** [comment]
With node 20 there is still a problem with the unit test, I haven't found out the cause yet, so I changed it to node 18 for now.

**Comment by KristjanESPERANTO** [comment]
@CFenner Is something still missing here? :slightly_smiling_face:

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "netatmo",
   "version": "2.0.0",
-  "description": "",
+  "description": "A module for the MagicMirror² to display informations about your rooms climate from your Netatmo system.",
```
Comment (CFenner) [inline package.json]:
```suggestion
  "description": "A module for the MagicMirror² to display information about your rooms climate from your Netatmo system.",
```

**Comment by CFenner** [comment]
@KristjanESPERANTO Can you have a look at the failing tests?

**Comment by KristjanESPERANTO** [comment]
> @KristjanESPERANTO Can you have a look at the failing tests?

I'll take a look at it, but as I see it, the test was already unsuccessful before this PR.

**Comment by KristjanESPERANTO** [comment]
I'm not very familiar with GitHub secrets, but it looks to me as if the test wants to access the API, but is not allowed to do so due to the missing token (HTTP 403 response). Did you save your token as secret in GitHub?

https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions

**Comment by CFenner** [comment]
🥵 all green beside the outdated token issue.

**Comment by KristjanESPERANTO** [comment]
Thanks for merging! As a side benefit, the module is now also displayed in full (with keywords) on [the new unofficial module list](https://kristjanesperanto.github.io/MagicMirror-3rd-Party-Modules/).

For now I can't help with the token issue :-(

**Comment by CFenner** [comment]
Nice overview, thanks!

**Pull request merged**